tests/Feature/CalendarTest.php

```diff
@@ -16,7 +15,8 @@ class CalendarTest extends TestCase
     public function it_shows_all_streams_in_calendar(): void
     {
         // Arrange
-        Stream::factory()->create(['title' => 'Stream #1', 'channel_title' => 'Channel 1', 'description' => 'Description 1', 'scheduled_start_time' => Carbon::yesterday(), 'youtube_id' => '1111', 'status' => StreamData::STATUS_NONE]);
+        Stream::factory()->create(['title' => 'Stream #0', 'channel_title' => 'Channel 0', 'description' => 'Description 0', 'scheduled_start_time' => Carbon::now()->subYears(2), 'youtube_id' => '0000']);
```

Comment:
Can you change the titles to make them more descriptive like `Stream two years old`?

And do we need to test the channel name as well? If not, please remove it because it should get filled automatically and we don't need to define it ourselves.

Reply:
https://github.com/christophrumpel/larastreamers/pull/34/commits/7d7b7c64a2cf03aa26b740deb4ee5da563d68057